faqs/atm7_http.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to enable HTTP on ATM7
```

Comment:
This seems too generic enough to be just an ATM7 thing, perhaps the title should be generic. The search can still have `atm7` but the content itself doesn't refer to it.